pyproject.toml

```diff
@@ -108,7 +108,7 @@ tests = [
   "pytest-servers[all]>=0.5.9",
   "pytest-benchmark[histogram]",
   "pytest-xdist>=3.3.1",
-  "pytest-env>=1.1.0",
+  "pytest-dotenv",
```

Comment:
Unrelated: fixing it here to be able to run tests agains CH locally (I'm using `.env.test` file for that)